src/font.c

```diff
@@ -2168,26 +2168,6 @@ font_score (Lisp_Object entity, Lisp_Object *spec_prop)
 }
 
 
-/* Concatenate all elements of LIST into one vector.  LIST is a list
-   of font-entity vectors.  */
-
-static Lisp_Object
-font_vconcat_entity_vectors (Lisp_Object list)
-{
-  EMACS_INT nargs = XFASTINT (Flength (list));
-  Lisp_Object *args;
-  USE_SAFE_ALLOCA;
-  SAFE_ALLOCA_LISP (args, nargs);
-  ptrdiff_t i;
-
-  for (i = 0; i < nargs; i++, list = XCDR (list))
-    args[i] = XCAR (list);
-  Lisp_Object result = Fvconcat (nargs, args);
-  SAFE_FREE ();
```

Comment:
This line here is where GC could be triggered. `SAFE_FREE` is a light wrapper around `unbind_to`.

Reply:
I think the only reason the `unbind_to` call is necessary is that the `SAFE_ALLOCA_LISP` invocation adds to the unwind protect stack. It should be fine without it. It's not a function that's invoked often.